### DISCUSSION TITLE
[SOLVED] The new browser template is not ideal

### DISCUSSION THREAD

**Discussion by God-damnit-all** [martinrotter/rssguard#592] [General]
Given I had to change all my post-process filter commands, I'd rather not downgrade to get a before and after, but this is what one of my feeds returns to me:

![image](https://user-images.githubusercontent.com/53661808/150002419-7e2e372f-c40d-45ff-bea4-181491a74bb3.png)

Not only is it really tiny for some reason, but, as I use the dark theme, the white background is an eyesore. The previous template was styled to match the rest of the theme nicely.

**EDIT**: Apparently zoom level can be controlled with ctrl+scroll, and it's persistent, so that solves that issue - although it's a bit weird that the default zoom level was so tiny to begin with.

**Comment by martinrotter** [comment]
1. You can zoom it.
2. There is "nudus-dark" variant of the skin + you can check "Force dark look" (Fusion only) to force full dark look.

**Reply by God-damnit-all** [reply]
> 1. You can zoom it.

Yeah, I ended up finding that, noted that in my edit.

> 2. There is "nudus-dark" variant of the skin + you can check "Force dark look" (Fusion only) to force full dark look.

I am already using that.
![image](https://user-images.githubusercontent.com/53661808/150032193-c0554b93-e83b-4afc-8209-e6a553c968bc.png)

**Reply by God-damnit-all** [reply]
> > 1. You can zoom it.
> 
> Yeah, I ended up finding that, noted that in my edit.
> 
> > 2. There is "nudus-dark" variant of the skin + you can check "Force dark look" (Fusion only) to force full dark look.
> 
> I am already using that. ![image](https://user-images.githubusercontent.com/53661808/150032193-c0554b93-e83b-4afc-8209-e6a553c968bc.png)

Wait. All of a sudden I'm getting the dark UI in the browser output when I wasn't getting it before... but I haven't touched those options at all since upgrading.

**Reply by martinrotter** [reply]
Define "All of a sudden".

The active "skin" is only applied after app restarts, that could be the thing which confused you?

**Reply by God-damnit-all** [reply]
> Define "All of a sudden".
> 
> The active "skin" is only applied after app restarts, that could be the thing which confused you?

No, the settings in my screenshot have been that way for a month at least.

The first time I started up after upgrading, the web browser had a white background. At some point, it started having a dark themed scheme to it, I don't know what changed or why it suddenly fixed itself.

**Reply by martinrotter** [reply]
> > 1. You can zoom it.
> 
> Yeah, I ended up finding that, noted that in my edit.
> 
> > 2. There is "nudus-dark" variant of the skin + you can check "Force dark look" (Fusion only) to force full dark look.
> 
> I am already using that. ![image](https://user-images.githubusercontent.com/53661808/150032193-c0554b93-e83b-4afc-8209-e6a553c968bc.png)

Btw I really found problem with "default font size" used for article viewer, now it should be much more sane default experience.

**Comment by akinokonomi** [comment]
> although it's a bit weird that the default zoom level was so tiny to begin with.

@ImportTaste, You can change font size in settings: Feeds & articles -> Articles (tab) -> Article browser font.
To restore zoom, click article viewer and press `Ctrl-0` (zero), to zoom in and out click article viewer and press either `Ctrl-+` or `Ctrl--`.

**Reply by martinrotter** [reply]
@akinokonomi

> although it's a bit weird that the default zoom level was so tiny to begin wit

I noticed it too, could you perhaps make change to nudus to default to bit bigger base font size?

**Reply by akinokonomi** [reply]
Nudus provides default font size of RSS Guard article browser (qt-webengine?). All it does is inherit that value.

Yes, I can increase it through `em`, but it will just make things inconsistent.

**Reply by martinrotter** [reply]
I see, in that case forget it, people can just zoom in and it is remembered.

**Reply by martinrotter** [reply]
Btw:

>All it does is inherit that value.

Yes, the "base" value is in fact specified by that font setting in.

**Reply by akinokonomi** [reply]
> I see, in that case forget it, people can just zoom in and it is remembered.

I wouldn't recommend that, because in that case article browser flickers when scrolling through articles list.

**Reply by martinrotter** [reply]
Indeed, but that is because of my low quality implementation.
